PORTING.md

```diff
@@ -1,7 +1,311 @@
-# Porting from OpenSSL/BoringSSL to AWS-LC
+# Porting from OpenSSL to BoringSSL
```

Comment:
Is this the original PORTING.md at main@f0558c3?

Reply:
Original PORTING.md is renamed to PORTING_TO_AWSLC.md. This is BoringSSL PORTING.

Reply:
Got it.